src/frontend/src/handler/variable.rs

```diff
@@ -48,6 +48,24 @@ pub fn handle_set(
     Ok(PgResponse::empty_result(StatementType::SET_VARIABLE))
 }
 
+pub(super) fn handle_set_time_zone(
+    handler_args: HandlerArgs,
+    value: SetTimeZoneValue,
+) -> Result<RwPgResponse> {
+    let tz_info = match value {
+        SetTimeZoneValue::Local => iana_time_zone::get_timezone()
+            .map_err(|e| ErrorCode::InternalError(format!("Failed to get local time zone: {}", e))),
+        SetTimeZoneValue::Default => Ok("UTC".to_string()),
+        SetTimeZoneValue::Literal(Value::DoubleQuotedString(s))
+        | SetTimeZoneValue::Literal(Value::SingleQuotedString(s)) => Ok(s),
+        _ => Ok(value.to_string()),
```

Comment:
`DoubleQuotedString` is not a PostgreSQL thing. It is considered an identifier instead:
```
dev=> set time zone "Asia/Shanghai";
ERROR:  QueryError: Invalid value ["\"Asia/Shanghai\""] for ["TIMEZONE"]
```
The `to_string` in wildcard match arm includes quotes as part of value unexpectedly. We should use `Ident::real_value` here.